src/views/GameSettingsWizard/index.tsx

```
@@ -41,31 +50,37 @@ export default function GameSettingsWizard({ close }) {
-    if (isPublicRoom(room)) return setStep(3);
+    if (isPublicRoom(room)) {
+      setStep(3);
+      return;
+    }
 
     setStep(2);
   }, [formData]);
 
-  const nextStep = (count) => {
+  const nextStep = (count?: number): void => {
     if (!Number.isInteger(count)) return setStep(step + 1);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)